news.css

```diff
@@ -0,0 +1,185 @@
+.top{
+    height: 20%;
+    width: 100%;
+    text-align: center;
+    background-color: #273852;
+
+}
+.box{
+    margin: 0.35%;
+    padding: 0.5% 1% 0.2% 1%;
+    border: 2px solid tomato;
+    color: #273852;
+    font-size: large;
+    display: inline-block;
+    background-color: #ffffff;
+}
+.category{
+    width: 20%;
+    height: 100%;
+    text-align: left;
+    
+}
+select{
+    width: 50%;
+    margin-top: 3%;
+    padding: 1%;
+}
+.subbtn,.email{
+    margin-top: 3%;

```

Comment:
the percentage is not recommended as it might break in small size resolution; for responsive page design please use flex and media queries;